### ISSUE TITLE
gcdserver scripts should use return codes (Trac #1982)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1974]
This in particular is so 'go' can determine if the configuration import was successful.

Migrated from https://code.icecube.wisc.edu/ticket/1982
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:15:18", 
    "description": "This in particular is so 'go' can determine if the configuration import was successful.", 
    "reporter": "jbraun", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067318169976", 
    "component": "other", 
    "summary": "gcdserver scripts should use return codes", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2017-04-13T18:07:34", 
    "milestone": "", 
    "owner": "jbraun", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20180607T19:48:26: olivas changed milestone from "December 2017" to "Autumnal Equinox 2018"

**Comment by icecube-trac** [comment]
Trac update at 20180923T08:01:47: 
* jbraun commented:

> In changeset 165705:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="165705"
> Add error return codes to import scripts.  Closes #1974.
> ```


* jbraun changed status from "new" to "closed"
* jbraun changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:15:18: 
* olivas changed milestone from "Autumnal Equinox 2018" to ""
* olivas commented:

> Milestone Autumnal Equinox 2018 deleted
